css/markdown-editor.styl

```diff
@@ -62,6 +62,7 @@
 .markdown-editor-help
   margin: 2em
   table
+    margin: auto
```

Comment:
Yeah I like that, looks better!  I think I properly updated, fixuped and forced push back, but let me know if not.